src/MapFlyer.ls

```diff
@@ -50,14 +50,23 @@ package
             if (stopped)
                 return;
 
+            var curTime = Platform.getTime();
+            var dt = Number(curTime - lastTime) / Number(1000);
```

Comment:
Yes, does it by elapsed time instead of assuming 60hz. I did this in part because the transition was very slow.